src/store/types.ts

```diff
@@ -0,0 +1,3 @@
+export const FETCH_REQUEST = 'FETCH_REQUEST';
+export const FETCH_SUCCESS = 'FETCH_SUCCESS';
+export const FETCH_ERROR = 'FETCH_ERROR';
```

Comment:
This file can be removed in favor of string literals. The `typesafe-actions` has good support and able to track any misspelling inside string literals. I mean, you can declare action types inside your action creators.

For example.

Instead of this
```JavaScript
const increment = createCustomAction(actions.INCREMENT);
```
you can do like this
```JavaScript
const increment = createCustomAction('INCREMENT');
```

Reply:
thanks) done